snap/quota/quota.go

```diff
@@ -1061,3 +1062,21 @@ func (s *QuotaGroupSet) AllQuotaGroups() []*Group {
 
 	return grps
 }
+
+// MakeServiceQuotaMap builds a map of services and their quota groups. The goal is
+// to make sure that if any quota group applies to a snap service, then we can easily
+// look it up in this based on it's snap.AppInfo. We take the snap instance name, the services
+// want to include in the map (should only be services of the snap name provided) and the quota
+// group of the snap.
+func MakeServiceQuotaMap(snapInfo *snap.Info, grp *Group) map[*snap.AppInfo]*Group {
```

Comment:
now that we moved it here I wonder if should become a method on Group:  GroupsForServices ?